frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerLifeCycle.java

```diff
@@ -38,10 +40,22 @@ public WorkerLifeCycle(ConfigManager configManager) {
     }
 
     public boolean startWorker(int port) {
-        String[] args = new String[3];
+        SocketAddress address = NettyUtils.getSocketAddress(port);
+        String[] args = new String[6];
         args[0] = "python";
         args[1] = "mms/model_service_worker.py";
-        args[2] = NettyUtils.getSocketAddress(port).toString();
+        args[2] = "--sock-name";
+        args[4] = "--sock-type";
+        if (address instanceof InetSocketAddress) {
+            args[5] = "tcp";
+            args[3] =
```

Comment:
It's better only pass port, let backend decide witch interface to bind on.

Reply:
Split it into host-port on the backend